.github/workflows/release.yml

```diff
@@ -93,10 +105,18 @@ jobs:
       - name: Run unit tests
         run: make test-coverage
 
-      - name: Build Helm Binaries
+      - name: Run Go Release
+        uses: goreleaser/goreleaser-action@v6.3.0
```

Comment:
as above (we need to pin to a specific release via hash (see other uses:). Please also include the semver version as a comment.)